front/src/reducers/__tests__/osrdconf.spec.ts

```diff
@@ -7,166 +7,231 @@ import {
   toggleOriginLinkedBounds,
 } from '../osrdconf';
 
-const createStore = (initialStateExtra?: Partial<OsrdConfState>) =>
-  createStoreWithoutMiddleware({ osrdconf: { ...initialState, ...initialStateExtra } });
+const simulationConfInitialState: Partial<OsrdConfState> = initialState.simulationConf;
+const stdcmConfInitialState = initialState.stdcmConf;
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const createStore = (initialStateExtra?: any) =>
+  createStoreWithoutMiddleware({
+    osrdconf: {
+      stdcmConf: stdcmConfInitialState,
+      mode: 'SIMULATION',
```

Comment:
```suggestion
      mode: MODES.SIMULATION,
```
Better use the defined constants

Reply:
yes